win32ss/user/ntuser/menu.c

```diff
@@ -6064,27 +6054,29 @@ NtUserGetMenuIndex(
    PMENU Menu, SubMenu;
    PITEM MenuItem;
    UINT i;
-   DECLARE_RETURN(UINT);
+   UINT Ret = 0xFFFFFFFF;
 
    TRACE("Enter NtUserGetMenuIndex\n");
    UserEnterShared();
 
    if ( !(Menu = UserGetMenuObject(hMenu)) ||
         !(SubMenu = UserGetMenuObject(hSubMenu)) )
-      RETURN(0xFFFFFFFF);
+      goto Exit;
 
    MenuItem = Menu->rgItems;
    for (i = 0; i < Menu->cItems; i++, MenuItem++)
    {
        if (MenuItem->spSubMenu == SubMenu)
-          RETURN(MenuItem->wID);
+       {
+          Ret = MenuItem->wID;
+          break;
```

Comment:
```suggestion
          goto Exit;
```
May be more safe if anything "dangerous" would ever be placed after the for-loop.

Reply:
I don't think there will be a problem if we want to add more code there in the future. We can check the `Ret` value before continue.

Reply:
Ok, not a biggy. Can live with that decision. Resolved.